packages/graphback-core/package.json

```diff
@@ -58,7 +59,7 @@
     "pluralize": "8.0.0"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0 || ^15.0.0",
-    "bson-ext": "2.0.5"
+    "bson-ext": "2.0.5",
```

Comment:
Should this be fixed, or should we allow a range of versions, like `2.x.x`?

Reply:
Done.